### PR TITLE
Create sambhavrana27.json'

### PR DESCRIPTION
**Pull request by sambhavrana27**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3838"></a>

### CODE REVIEW

**Review by Tejender1521 — APPROVED** [review]
Everything looks good to me

**Comment by sambhavrana27** [comment]
Thanks, Mr. Upadhyay !!!  Means a lot .From: TEJENDER UPADHYAYSent: 23 January 2023 07:25 PMTo: EddieHubCommunity/LinkFreeCc: Sambhav Rana; MentionSubject: Re: [EddieHubCommunity/LinkFree] Create sambhavrana27.json' (PR ***@***.*** approved this pull request.Everything looks good to me—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Review by kumarsonsoff3 — DISMISSED** [review]
Please move your file to data directory. 

Change the filename from `'data/sambhavrana27.json'`  - >  `data/sambhavrana27.json`

**Review by devvsakib — CHANGES_REQUESTED** [review]
you can add tags like this:
image

File: 'data/sambhavrana27.json' [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "links": [
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/sambhavnrana/?hl=en",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/sambhav-rana-b03558229/",
+      "icon": "FaLinkedIn"
```
Comment (devvsakib) [inline 'data/sambhavrana27.json':17]: icon name: FaLinkedin

**Review by devvsakib — CHANGES_REQUESTED** [review]

**Pull request merged**